lib/go-tc/deliveryservice_requests.go

```diff
@@ -435,6 +435,58 @@ type DeliveryServiceRequestNullable struct {
 	XMLID           *string                     `json:"-" db:"xml_id"`
 }
 
+// Downgrade will convert an instance of DeliveryServiceRequestV41 to DeliveryServiceRequestV40
```

Comment:
GoDoc should end with punctuation

Reply:
Also, it'd be helpful to note that the Requested and/or Original DSes are shallow-copied in this case.